examples/src/index.css

```diff
@@ -6,12 +6,18 @@ body {
 .message {
   background-color: #dddddd;
 }
-.message-container {
+.box-container {
   border: 1px solid #888888;
   width: 600px;
   margin:0 auto;
   padding: 5px;
 }
 .App {
   text-align: center;
+}
+.active {
```

Comment:
Perhaps you wanted to use this in the links?

Reply:
As mentioned in the comment above, it's used by the `NavLink`'s default active class name